static/js/vue-cdr-access/src/components/browseDisplay.vue

```diff
@@ -66,26 +67,37 @@
 
         watch: {
             '$route.query'(d) {
-                this.retrieveData();
+                this.browseTypeFromUrl(this.is_admin_unit);
+                if (!this.is_page_loading && (this.applicablePage)) {
+                    this.retrieveData();
+                }
             }
         },
 
-        mixins: [routeUtils],
+        mixins: [routeUtils, browseOptionUtils],
 
         data() {
             return {
                 column_size: 'is-3',
                 container_name: '',
                 container_metadata: {},
+                default_work_type: 'Work',
+                is_admin_unit: false,
                 is_collection: false,
                 is_folder: false,
+                is_page_loading: true,
                 record_count: 0,
                 record_list: [],
+                search_method: 'searchJson',
                 uuid: ''
             }
         },
 
         computed: {
+            applicablePage() {
```

Comment:
Would it be possible to only include this component on pages where it is applicable, versus including it on all record pages and then disabling it? Or do you feel like it's better in the long run to do things this way?

Reply:
Not sure how we'd tell it's an applicable page otherwise. It matching the route /record/<uuid> in the Vue router, so it's going to think it needs to be loaded. As far as I know nothing in the url differentiates between a say a file vs a collection page.